### PR TITLE
[TwigBridge] Remove usage of `Node()` instantiations

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Issues        | n/a
| License       | MIT

See https://github.com/twigphp/Twig/pull/4335 (which should be merged first)

### CODE REVIEW

**Review by alexandre-daubois — APPROVED** [review]

**Comment by xabbuh** [comment]
@fabpot Can you please rebase to trigger the pipeline?

**Comment by fabpot** [comment]
> @fabpot Can you please rebase to trigger the pipeline?

Done

File: src/Symfony/Bridge/Twig/NodeVisitor/TranslationDefaultDomainNodeVisitor.php [hunk 1]
```diff
@@ -53,7 +54,7 @@ public function enterNode(Node $node, Environment $env): Node
                 $name = new AssignNameExpression($var, $node->getTemplateLine());
                 $this->scope->set('domain', new NameExpression($var, $node->getTemplateLine()));
 
-                return new SetNode(false, new Node([$name]), new Node([$node->getNode('expr')]), $node->getTemplateLine());
+                return new SetNode(false, new Nodes([$name]), new Nodes([$node->getNode('expr')]), $node->getTemplateLine());
```
Comment (xabbuh) [inline src/Symfony/Bridge/Twig/NodeVisitor/TranslationDefaultDomainNodeVisitor.php]: All these usages either need to happen conditionally (when the `Nodes` class exists) or we need to bump the Twig requirement. But I guess dropping Twig 2 support in a patch release is not an option.

**Review by xabbuh — CHANGES_REQUESTED** [review]

File: src/Symfony/Bridge/Twig/NodeVisitor/TranslationDefaultDomainNodeVisitor.php (hunk 1 above)
Comment (fabpot) [reply src/Symfony/Bridge/Twig/NodeVisitor/TranslationDefaultDomainNodeVisitor.php]: Done

**Review by xabbuh — APPROVED** [review]

**Pull request merged**